modules/commons/src/test/java/org/apache/synapse/commons/json/JsonStreamingBuilderTest.java

```diff
@@ -19,8 +19,36 @@
 package org.apache.synapse.commons.json;
 
 import junit.framework.TestCase;
+import org.apache.axiom.om.OMElement;
+import org.apache.axis2.builder.Builder;
+import org.apache.axis2.context.MessageContext;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import java.io.InputStream;
 
 public class JsonStreamingBuilderTest extends TestCase {
     public void testCase() {
+        testInvalidJson(invalidJson);
+    }
+
+    private Builder jsonBuilder = Util.newJsonStreamBuilder();
+    private static final Log log = LogFactory.getLog(JsonStreamingBuilderTest.class);
+
+    private static final String invalidJson = "{\n" +
+            "\"account_number\":\"1234567890\",\n" +
+            "\"routing_number\":\"09100001\n" +
+            "\"image_type\":\"COMMERCIAL_DEPOSIT\"\n" +
+            "}";
+
+    public  void testInvalidJson(String jsonIn) {
+        try {
+            MessageContext message = Util.newMessageContext();
+            InputStream inputStream = Util.newInputStream(jsonIn.getBytes());
+            OMElement element  = jsonBuilder.processDocument(inputStream, "application/json", message);
+            message.getEnvelope().getBody().addChild(element);
+            log.info(message.getEnvelope().getBody().toString());
+        } catch (Exception e) {
+            assertEquals(e.getMessage(),"Error: could not match input");
```

Comment:
Are we testing whether the runtime exception is thrown correctly? if so we can check instance of RuntimeException